openfpga/src/fpga_bitstream/build_device_bitstream.cpp

```diff
@@ -230,6 +230,12 @@ BitstreamManager build_device_bitstream(const VprContext& vpr_ctx,
     openfpga_ctx.vpr_device_annotation(), openfpga_ctx.vpr_routing_annotation(),
     vpr_ctx.device().rr_graph, openfpga_ctx.device_rr_gsb(),
     openfpga_ctx.flow_manager().compress_routing(), verbose);
+
+  /* Apply path bit value */
```

Comment:
Create another function like ``overwrite_bitstream()`` and place it in a separated file. 
The ``build_device_bitstream()`` should have only one purpose: decode bitstream based on VPR results.
You can call the ``overwrite_bitstream()`` at

https://github.com/lnis-uofu/OpenFPGA/blob/aed082817ed3421ac0cfc45e305ba9191bf457ad/openfpga/src/base/openfpga_bitstream_template.h#L62

Reply:
Done via https://github.com/chungshien/OpenFPGA/commit/2a3d69aded840c54212dd9652670ab54e9ab1222